R/ari_narrate.R

```diff
@@ -47,26 +50,40 @@
 #'             voice = "Joey")
 #' 
 #' }
-ari_narrate <- function(script, slides, output = "output.mp4", voice,
+ari_narrate <- function(script, slides, 
+                        output = tempfile(fileext = ".mp4"),
```

Comment:
I don't disagree, but it will default to creating to overwriting a file.  Also, helps with CRAN because never writes anywhere it shouldn't in the `R CMD check`

Reply:
It was passing R CMD check previously though, right?

Reply:
Yes - nothing was an issue here because CRAN wasn't actually running anything (no `ffmpeg` or `Polly`).  Don't like the behavior of changing directory or disk data unless explicit from user.  OK if you reject this, but def change the example to make sure that wherever you're running the check doesn't pop out an `output.mp4` in that directory (aka put the `tempfile` code in the example above).